### PR TITLE
Blazor Todo list tutorial updates

### LINKED ISSUE #19045: Remove duplicate and unnecessary content from the to do list tutorial
Much of the content in this article is duplicated form the [new Blazor getting started tutorial](https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro). The proposal is to remove the duplicated content and focus this article on just the to do list tutorial.

@guardrex 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 36e7dc3c-54c4-35de-203e-d230869e8417
* Version Independent ID: dc5b52d5-c8c6-1440-6b3a-fda898167d0c
* Content: [Build your first Blazor app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/build-your-first-blazor-app?view=aspnetcore-3.1)
* Content Source: [aspnetcore/tutorials/build-your-first-blazor-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/build-your-first-blazor-app.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #19045

**Moved from https://github.com/dotnet/AspNetCore.Docs/pull/19050.**

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/build-blazor-app?view=aspnetcore-3.0&branch=pr-en-us-19053)

* I changed the title/URL because the new Get Started tutorial uses the 'build your first' language.
* I ultimately decided to draft this with a "build-a-blazor-app" URL segment (file name).
* By **_not_** using "todo" in the URL (file name) and UID, we have the greatest flexibility to replace the lovely todo experience. :smile:

### CODE REVIEW

File: aspnetcore/tutorials/build-a-blazor-app.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+author: guardrex
+description: Build a Blazor app step-by-step.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/02/2020
+no-loc: [Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: tutorials/build-a-blazor-app
+---
+# Build a Blazor app
```
Comment (danroth27) [inline aspnetcore/tutorials/build-a-blazor-app.md:12]:
```suggestion
# Build a Blazor Todo List app
```

File: aspnetcore/tutorials/build-a-blazor-app.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+> [!div class="checklist"]
+> * Create a todo list Blazor app project
+> * Modify Razor components
+> * Use event handling and data binding in components
+> * Use dependency injection (DI) and routing in a Blazor app
+
+At the end of this tutorial, you'll have a working todo list app.
+
+1. Follow the guidance in the [Build your first Blazor app tutorial](https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro) to create a Blazor project. Name the project `ToDoList`.
```
Comment (danroth27) [inline aspnetcore/tutorials/build-a-blazor-app.md:26]:
```suggestion
1.Create a Blazor project named `TodoList`.

    ```
    dotnet new blazorserver -o TodoList
    cd TodoList
    ```
```

File: aspnetcore/tutorials/build-a-blazor-app.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+1. Add a new `Todo` Razor component to the app in the `Pages` folder. If you're using Visual Studio, right-click the `Pages` folder and select **Add** > **New Item** > **Razor Component**. Name the component's file `Todo.razor`. In other development environments, add a blank file to the `Pages` folder named `Todo.razor`. Razor component file names require a capitalized first letter, so confirm that the `Todo` component file name starts with a capital letter `T`.
```
Comment (danroth27) [inline aspnetcore/tutorials/build-a-blazor-app.md:28]:
```suggestion
1. Add a new `Todo` Razor component to the app in the `Pages` folder. 

    ```
    dotnet new razorcomponent -n Todo -o Pages
    ```

    > Important: Razor component file names require a capitalized first letter, so confirm that the `Todo` component file name starts with a capital letter `T`.
```

File: aspnetcore/tutorials/build-a-blazor-app.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+1. Provide the initial markup for the component:
```
Comment (danroth27) [inline aspnetcore/tutorials/build-a-blazor-app.md:30]:
```suggestion
1. In *Pages/Todo.razor* provide the initial markup for the component:
```

File: aspnetcore/tutorials/build-blazor-app.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+author: guardrex
+description: Build a Blazor app step-by-step.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/02/2020
+no-loc: [Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: tutorials/build-blazor-app
+---
+# Build a Blazor app
```
[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/tutorials/build-blazor-app.md]: I don't think it's a proper noun.

Do you want "Todo List" for sure, or go with "todo list"?

**Comment by guardrex** [comment]
@danroth27 ... I think that licks that round of updates.

**Comment by guardrex** [comment]
..... except for the folder name. 🙈

*It should be ok now.* ........ :point_left: Put that on my tombstone plz! ⚰️

File: aspnetcore/tutorials/build-blazor-app.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (danroth27) [reply aspnetcore/tutorials/build-blazor-app.md]: Sorry, todo list is fine.

**Comment by guardrex** [comment]
@scottaddie Plz **_post review_** this time ... I need to get unblocked here for the next round of updates.

**Comment by guardrex** [comment]
One thing @scottaddie ... I just noticed a problem that I'll fix in the index.yml file. I'll square that away on the next PR coming shortly.

**Pull request merged**